app/views/pdftemplates/rulingPdf.scala.html

```diff
@@ -103,7 +103,7 @@ <h2>@messages("pdf.ruling.section.ruling.heading")</h2>
                 </p>
                 <p>
                     <span class="section-label">@messages("pdf.ruling.section.ruling.rulingReference.label"):</span>
-                    @messages("caseReference.format", c.reference)
+                    GB@messages("caseReference.format", c.reference)
```

Comment:
Similar here. Has Joe said it should start with GB?

Reply:
After post-amigos this has been removed now, so no longer a thing